api/src/main/java/org/openmrs/module/ohricore/engine/CommonsUUID.java

```diff
@@ -0,0 +1,15 @@
+package org.openmrs.module.ohricore.engine;
+
+/**
+ * @author MayanjaXL, Amos, Stephen, smallGod date: 30/06/2021
+ */
+public final class CommonsUUID {
```

Comment:
What are commons? This is not clear, also use an Interface instead of a final Class for holding constants then you can implement them to access the constants

Reply:
Actually I had done an interface at first and the team (with Fitti) asked me to use a class instead. That I was misusing the interface which should only hold behaviour that needs to be implemented.
I was pointed to the OpenMRS way of doing things like this class in the package org.openmrs.util for constants;

public final class OpenmrsConstants {
	
	private static final Logger log = LoggerFactory.getLogger(OpenmrsConstants.class);
	
	public static String KEY_OPENMRS_APPLICATION_DATA_DIRECTORY = "OPENMRS_APPLICATION_DATA_DIRECTORY";
	
	public static final int CONCEPT_CLASS_DRUG = 3;
	
	----
}

Now on the name CommonsUUID, Can I re-name this to "CommonConceptAnswersUUID" ?

Reply:
@smallgod You need to be more specific, why are you calling them common?

Reply:
@smallgod Why are you calling them common, class names have to be very specific and clear - if you call it common then you have to put all the concepts constants there